frontend/integration-tests/tests/kubevirt/utils.ts

```diff
@@ -15,3 +37,20 @@ export function removeLeakedResources(leakedResources: Set<string>){
       });
   }
 }
+
+export async function selectDropdownOption(dropdownId: string, option: string) {
+  if (dropdownId.startsWith('#network') || dropdownId.startsWith('#storage')) {
+    // workaround for Bug that namespace dropdown needs to be clicked twice, remove when resolved
```

Comment:
actually this is more about the order of operations. The dropdown can be opened with one click only when it is the first operation. 

But if you click on input (inside table) before that, then you need to click twice. I.e. when the table input gains focus.

Dunno if it makes sense to include this somehow into the API.

Btw, this will be fixed once we move to PF4 tables.

Reply:
I think it makes sense to hide this detail from the tests itself. Handling of this "feature" here sounds best to me.
Moreover, if the issue will be gone in PF4 tables

Reply:
@suomiy Ah, you're right. So we can get rid of this condition if I just change the order in the respective methods in  ```models/wizard.ts ```. Would you mind me addressing this in a follow up PR?

Reply:
@suomiy included the change here.

Reply:
ok, can you just include a comment about this issue before `await wizardView.selectTableDropdownAttribute` ?

maybe you can reference the https://github.com/kubevirt/web-ui-components/issues/9 issue?

Reply:
Sure, done.